cloud-sql/postgres/knex/test/createTable.test.js

```diff
@@ -22,18 +22,28 @@ const {exec} = require('child_process');
 const cwd = path.join(__dirname, '..');
 
 const {DB_USER, DB_PASS, DB_NAME} = process.env;
-const CONNECTION_NAME = process.env.CLOUD_SQL_CONNECTION_NAME;
-
+const CONNECTION_NAME = process.env.INSTANCE_CONNECTION_NAME;
+const DB_HOST = process.env.DB_HOST;
```

Comment:
nit: Why not include this in the destructuring statement above?

Reply:
updated in latest commit